packages/vue/src/components/primitives/base-heading.vue

```diff
@@ -12,16 +12,21 @@ export default defineComponent({
   setup(props, { slots, attrs }): Record<string, unknown> | (() => unknown) {
     const defaultSlot = slots.default ? slots.default() : [];
     const headingI = slots.headingI ? slots.headingI() : [];
+    const headerClass = `${attrs.class || ''} amplify-heading--${attrs.level}`;
```

Comment:
This is a little confusing on Vue's part but the `level` is passed in as a prop, not attrs. `attrs` only includes non prop attributes passed into the component.
```suggestion
    const headerClass = `${attrs.class || ''} amplify-heading--${props.level}`;
```

Reply:
good catch, thanks that has been updated